automation/BuildRegistrationDefinition/.gitignore

```diff
@@ -0,0 +1,2 @@
+/default_fhe_user.pri
+/default_fhe_user.pub
```

Comment:
I asked @boland25 if this should not be *.pri and *.pub. I would hate to see someone use a non-default user and somehow commit their keys accidentally.

Reply:
yes it should be *.pri and *.pub good catch